spec/integration/manifest.spec.yml

```diff
@@ -101,6 +101,21 @@ spec:
               default: basic
               examples:
               - basic
+            capabilities:
+              description: |-
+                Stack features that are required by the package to work properly.
+                The package should not be used in deployments without the indicated features.
+                Packages that don't indicate any capability condition can be used on any deployment.
+              type: array
+              items:
+                type: string
+                enum:
+                  - apm
+                  - enterprise_search
+                  - observability
+                  - security
+                  - serverless_search
+                  - uptime
```

Comment:
Maybe it is worth to try to add `uniqueItems: true` field here (https://json-schema.org/understanding-json-schema/reference/array.html#uniqueness), to ensure that there are no duplicated items.
```suggestion
              type: array
              uniqueItems: true
              items:
                type: string
                enum:
                  - apm
                  - enterprise_search
                  - observability
                  - security
                  - serverless_search
                  - uptime
```

Never tested in any field this setting, but it could be helpful even with those few elements. WDYT ?

Tested with this change
```diff
--- test/packages/good_v2/manifest.yml
+++ test/packages/good_v2/manifest.yml
@@ -11,6 +11,9 @@ conditions:
   elastic.subscription: 'basic'
   elastic.capabilities:
     - observability
+    - observability
+    - apm
+    - apm
 vars:
   - name: package_password
     type: password
```
shows these errors:
```
        	Error Trace:	/home/mariorodriguez/Coding/work/package-spec/code/go/pkg/validator/validator_test.go:599
        	Error:      	Received unexpected error:
        	            	found 2 validation errors:
        	            	   1. file "../../../../test/packages/good_v2/manifest.yml" is invalid: field conditions.elastic.capabilities: array items[0,1] must be unique
        	            	   2. file "../../../../test/packages/good_v2/manifest.yml" is invalid: field conditions.elastic.capabilities: array items[2,3] must be unique
        	Test:       	TestValidateRoutingRules/good_v2
```